Samples/InpaintHTTP/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Nancy" version="2.0.0" targetFramework="net461" />
+  <package id="Nancy.Hosting.Self" version="2.0.0" targetFramework="net461" />
+  <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net461" />
+</packages>
```

Comment:
The following packages are missing. The project doesn't compile without them.

- Accord
- Accord.MachineLearning
- Accord.Math
- Accord.Statistics
- Zavolokas.GdiExtensions
- Zavolokas.ImageProcessing.PatchMatch
- Zavolokas.Math
- Zavolokas.Structures
- Zavolokas.Structures.Gdi

Reply:
Forgot to clean up my mess with the packages, will fix that asap!